reV/handlers/exclusions.py

```diff
@@ -144,8 +144,7 @@ def layers(self):
         layers : list
             List of exclusion layers
         """
-        layers = [ds for ds in self.h5
-                  if ds != 'meta']
```

Comment:
No meta data in exclusion files? Do we not have to filter out the lat/lon datasets?

Reply:
I was surprised as well but makes sense as instead of meta we have a latitude and longitude dataset